src/membership.md

```diff
@@ -37,17 +37,20 @@ These are the questions we ask ourselves when deciding whether someone would be
 
 * Has this person demonstrated **strong language design skills**?
     * Have they led an impactful initiative to completion?
-    * Have they identified a criticial flaw in someone else's design that led to the design being altered?
```

Comment:
I don't know if I want to reduce this down to only the "leading to completion" part. What about something like

> Are they able to identify flaws in a design and, just as importantly, come up with creative solutions?

Though I'm not sure if that hits the mark entirely. "Creative" could be interpreted in a lot of ways. It's possible to come up with something creative that's also very complex, for example.

Reply:
Hmm. Yes, I like that. I'm also thinking that "leading to completion" isn't *necessarily* key -- I feel like it's ok to drive an important thing that winds up being abandoned, as long as that is actively decided, and the work in the interim feels good.